variables.tf

```diff
@@ -76,6 +76,11 @@ variable "services_delete_on_termination" {
   default     = "false"
 }
 
+variable "destroy_s3_bucket" {
```

Comment:
`force_destroy_s3_bucket`?

Reply:
May be named poorly?  The idea is that it will delete an S3 bucket even if it has files in it.

Reply:
Ah sorry about that, yeah, I was suggesting a new name, I'll be more verbose in future.

I think `force_destroy_s3_bucket` is clearer, because it will attempt to delete the bucket regardless of this setting.